### PR TITLE
NOISSUE - Scale up nats, envoy, authn, things and mqtt

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milošević <iva@blokovi.com>

### CODE REVIEW

File: charts/mainflux/README.md [hunk 1]
```diff
@@ -20,13 +20,16 @@ The following table lists the configurable parameters and their default values.
...
+| `defaults.replicaCount`                | Replicas of MQTT adapter, Things, Envoy and Authn                          | `3`            |
 | `nginx_internal.mtls.tls`              | TLS secret which contains the server cert/key                              | `''`           |
 | `nginx_internal.mtls.intermediate_crt` | Generic secret which contains the intermediate cert used to verify clients | `''`           |
 | `ingress.enabled`                      | Should the Nginx Ingress be created                                        | `true`         |
 | `ingress.hostname`                     | Hostname for the Nginx Ingress                                             | `''`           |
 | `ingress.tls.hostname`                 | Hostname of the Nginx Ingress certificate                                  | `''`           |
 | `ingress.tls.secret`                   | TLS secret for the Nginx Ingress                                           | `''`           |
+| `nats.maxPayload`                      | Maximum payload size in bytes that the NATS server will accept             | `268435456`    |
```
Comment (drasko) [inline charts/mainflux/README.md:31]: Is this one used?

Comment (blokovi) [reply charts/mainflux/README.md:31]: I just added this to readme, it was already as default values (it's same value we got [docker composition](https://github.com/mainflux/mainflux/blob/master/docker/nats/nats.conf))

Comment (manuio) [reply charts/mainflux/README.md:31]: Do we need to highlight values inside the table? Do we also need `''`?

Comment (blokovi) [reply charts/mainflux/README.md]: Removed

**Pull request merged**